pom.xml

```diff
@@ -48,18 +48,19 @@
         <dependency.freemarker.version>2.3.28</dependency.freemarker.version>
         <dependency.rhino.version>1.7.11</dependency.rhino.version>
         <dependency.httpcomponents.version>4.5.9</dependency.httpcomponents.version>
-        <dependency.alfresco-repository.version>7.185</dependency.alfresco-repository.version>
+        <dependency.alfresco-repository.version>7.193</dependency.alfresco-repository.version>
```

Comment:
Invalid alfresco-remote-api dependency 7.193 (I see it's correctly redeclared below, i.e.: 7.144).